fs_storage/models/fs_storage.py

```diff
@@ -454,7 +454,8 @@ def delete(self, relative_path) -> None:
 
     def action_test_config(self) -> None:
         try:
-            self._check_connection(self.__fs)
+            # pylint: disable=W0104
+            self.fs
```

Comment:
A comment is welcome

Reply:
@sebalix I have added a change to clear the slot when the connection fails, maybe this fixes the issue where the FS object was being kept although the connection was closed